functions/java-jre.bats

```diff
@@ -7,4 +7,13 @@ load helpers
   run java -version
   [ "$status" -eq 0 ]
   [[ $output == *"Zulu"* ]]
-}
\ No newline at end of file
+  echo -e "# \e[32mInstallation of Zulu java verified." >&3 
+}
+
+
+@test "installation-java_new" {
```

Comment:
Don't understand what new parts this actually test as it installs exactly the same content as before?
Okey, it kind of test to install a new java version ontop of the other one.

To add to it I have made a complete rewrite of the java script including tests so this code will be obsolete: https://github.com/openhab/openhabian/pull/675/files

Reply:
removed